frontend/src/FullPageRoutes/Matching.tsx

```diff
@@ -75,8 +76,23 @@ function MatchControls({
   onReject,
   onReport,
 }: MatchControlsType): JSX.Element {
+  const [showReport, setShowReport] = React.useState<boolean>(false)
+  const [description, setDescription] = React.useState<string>("")
   return (
     <Grid item container direction="row" justify="space-around">
+      <AbstractModal
+        title="Report"
+        description="Optional report message"
+        isModalOpen={showReport}
+        handleConfirm={() => onReport(matchId, description)}
+        handleCancel={() => setShowReport(false)}
```

Comment:
should the description be set to an empty string on cancel or at least after a match has been accepted/rejected?